src/orchestration/Orchestration.ts

```diff
@@ -402,7 +402,8 @@ export class Orchestration {
                 id: applicationId,
                 version: applicationVersion
             },
-            this.config
+            this.config,
+            this.eventBus
```

Comment:
Callout: this was a bug that was included in #445. Currently implementing robust tests to prevent this in the futrue